desktop/org.yorba.geary.gschema.xml

```diff
@@ -1,73 +1,85 @@
 <schemalist>
 
 <schema id="org.yorba.geary" path="/org/yorba/geary/">
-    
+
     <key name="window-maximize" type="b">
         <default>false</default>
         <summary>maximize window</summary>
         <description>True if library application is maximized, false otherwise.</description>
     </key>
-    
+
     <key name="window-width" type="i">
         <default>1024</default>
         <summary>width of window</summary>
         <description>The last recorded width of the application window.</description>
     </key>
-    
+
     <key name="window-height" type="i">
         <default>768</default>
         <summary>height of window</summary>
         <description>The last recorded height of the application window.</description>
     </key>
-    
+
     <key name="folder-list-pane-position" type="i">
         <default>100</default>
         <summary>position of folder list pane</summary>
         <description>Position of the folder list Paned grabber.</description>
     </key>
-    
+
     <key name="folder-list-pane-position-horizontal" type="i">
         <default>-1</default>
         <summary>position of folder list pane when horizontal</summary>
         <description>Position of the folder list Paned grabber in the horizontal orientation.</description>
     </key>
-    
+
     <key name="folder-list-pane-position-vertical" type="i">
         <default>200</default>
         <summary>position of folder list pane when vertical</summary>
         <description>Position of the folder list Paned grabber in the vertical orientation.</description>
     </key>
-    
+
     <key name="folder-list-pane-horizontal" type="b">
         <default>true</default>
         <summary>orientation of the folder list pane</summary>
         <description>True if the folder list Paned is in the horizontal orientation.</description>
     </key>
-    
+
+    <key name="compress-conversation" type="b">
```

Comment:
I'm not really sure what type indicates, should this be something else? Also for hide-conversation.